tests/simulation/translator/test_output_translation.py

```diff
@@ -403,11 +403,17 @@ def probe_output_config():
         {
             "monitors": {
                 "prb 10": {
+                    "animationFrequency": 1,
```

Comment:
Can you add a test where `TimeAverageProbeOutput` & `TimeAverageSurfaceProbeOutput` are used along with Non-average counterparts? Just to make sure the merge in translator works.

Reply:
Done